operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -333,16 +328,47 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
     }
 
     /**
-     * @return the list of patient subresources that will be included in the $everything operaetion
+     * @return the list of patient subresources that will be included in the $everything operation
      * @throws FHIRSearchException
      */
     private List<String> getDefaultIncludedResourceTypes() throws FHIRSearchException {
         List<String> resourceTypes = new ArrayList<>(CompartmentUtil.getCompartmentResourceTypes(PATIENT));
-        // TODO: Practitioner and Organization are not included in the getCompartmentReourceTypes() by default but it seems
-        // like a couple of good additional resources to include and they are even mentioned as examples of resources
-        // to include in the docs: https://www.hl7.org/fhir/operation-patient-everything.html
-        // resourceTypes.add(Practitioner.class.getSimpleName());
-        // resourceTypes.add(Organization.class.getSimpleName());
+        
+        try {
+            List<String> supportedResourceTypes = FHIRConfigHelper.getSupportedResourceTypes();
```

Comment:
maybe filter this list down to just the resource types that support the `search` interaction?
I think you pass the resourceHelper from the doInvoke method and then call `FHIRRestHelper.validateInteraction(Interaction.SEARCH, resourcesType)`